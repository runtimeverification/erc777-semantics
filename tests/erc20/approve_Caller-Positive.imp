test.preamble();

// Testing approve(Spender,Allowance) when Spender = caller and Allowance > 0.
// Approving allowance to yourself is needed in order to allow calls to
// transferFrom from yourself.

test.approve(7, 37);
