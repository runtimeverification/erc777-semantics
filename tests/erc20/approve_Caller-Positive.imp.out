7 is approving allowance 37 for 7
